src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,86 @@
-export const Pagination = () => {};
+import React from 'react';
+import classNames from 'classnames';
+
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: number,
+  PerPage: number,
+  currentPage: number,
+  onPageChange: (value: number) => void,
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  PerPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const paginateItemsCount = Math.ceil(total / PerPage);
+  const paginateItemsArray = getNumbers(1, paginateItemsCount);
+
+  return (
+    <ul className="pagination">
+      <li
+        className={classNames(
+          'page-item',
+          {
+            disabled: currentPage === 1,
+          },
+        )}
+      >
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={currentPage === 1 ? 'true' : 'false'}
+          onClick={() => {
+            onPageChange(currentPage - 1);
+          }}
```

Comment:
make handler in component which will be take page and return arrow function. and just call her and in other places.
const makeOnPageChange = (page) => () => onPageChange(page);

```suggestion
          onClick={makeOnPageChange(currentPage - 1)};
```